test/hotspot/jtreg/compiler/c2/Test8263352.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
We don't use bug ID for tests anymore. Rename it to something meaningful and move it to corresponding `compiler/loopopts/` or `compiler/loopstripmining/` directory.